### DISCUSSION TITLE
Custom Asset Paths Issue with Vite and Nitro in Nuxt 3

### DISCUSSION THREAD

**Discussion by danyal14** [nuxt/nuxt#29072] [Questions]
I’m currently configuring a Nuxt 3 project to use custom asset paths with _nuxt_v3/ for both Vite and Nitro. While Vite is correctly generating files in .nuxt/dist/**/_nuxt_v3/, Nuxt Nitro is still creating files in .output/public/_nuxt.

My goal is to ensure that assets are consistently served from the _nuxt_v3/ directory across both Vite and Nitro.

Here is my current nuxt.config.ts configuration:

```
export default defineNuxtConfig({
  buildDir: ".nuxt",
  app: {
    buildAssetsDir: "_nuxt_v3/",
    rootId: "_nuxt_v3",
    id: "_nuxt_v3",
  },
  build: {
    publicPath: "/_nuxt_v3/",
  },
  vite: {
    build: {
      rollupOptions: {
        output: {
          chunkFileNames: "_nuxt_v3/[name].[hash].js",
          entryFileNames: "_nuxt_v3/[name].[hash].js",
          assetFileNames: "_nuxt_v3/[name].[hash].[ext]",
        },
      },
    },
  },
  nitro: {
    staticAssets: {
      dir: 'public/_nuxt_v3/'
    },
  },
})
```

Despite these settings, Nitro is still using the default _nuxt path and attempts to fetch resources from paths like /_nuxt/_nuxt_v3/error-404.CHiphUsI.js.

I have thoroughly reviewed the Nuxt documentation and tried various configurations but haven’t been able to resolve this issue.

Could someone provide guidance on how to align the asset paths for both Vite and Nitro or suggest any additional configurations required to achieve this?

Thank you in advance for your help!

**Comment by xjccc** [comment]
```
nitro: {
    output: {
      dir: '_nuxt_v3',
      publicDir: '_nuxt_v3'
   }
}
```
